README.md

```diff
@@ -115,13 +115,13 @@ In the illustration above the structure of the control system an its supporting
 
 #### Global Navigator
 
-The purpose of the global navigator is to find a path the robot should follow to reach its goal. The the goal points is set manually and when the robot reaches it, a new one will be chosen by the operator. It will use information from the global map, a map of the combined scans from all robots. This is implemented using the already existing ROS package move_base that calculates a global path for the robot using a form of A-star algorithm. 
+The purpose of the global navigator is to find a path the robot should follow to reach its goal. The the goal points are set manually and when the robot reaches it, a new one will be chosen by the operator. It will use information from the global map, a map of the combined scans from all robots. This was implemented using the already existing ROS package move_base that calculates a global path for the robot using Dynamic Window Approach Planner algorithm. 
```

Comment:
Dynamic Window Approach Planner is for Local Navigation while Global_Planner is for global plans. Global_Planner uses Dijkstras algorithm.

Reply:
Actually it seems I am wrong about this. It is the /move_base/DWAPlannerROS/global_plan that is used and not the /move_base/NavfnROS/plan.

Reply:
du får fixa detta själv då så att blir rätt